tailwind.config.js

```diff
@@ -122,6 +125,9 @@ module.exports = {
       veryLightBlue: '#E1EBFF',
       veryLightBlue2: '#E2EBFF',
       evenLighterBlue: '#ECF2FF',
+      // new blueish colors
```

Comment:
Comments like this go out of date quickly, so it's better to add comments like this onto the PR itself (like I'm doing here) rather than adding them into the codebase.